client/app/components/Story/StoryActions.jsx

```diff
@@ -40,6 +40,7 @@ export type Actions = {
 
 export type Props = {
   actions: Actions,
+  storyName: string,
```

Comment:
You'll want to pass in `storyName` as a prop in all use cases of `StoryActions`. As raised in the Flow failure in the [recent CI build](https://app.circleci.com/pipelines/github/ifmeorg/ifme/1913/workflows/d764a95f-2e6f-4118-a507-c0b114291060/jobs/15457), we're missing the prop in https://github.com/ifmeorg/ifme/blob/main/client/app/widgets/Comments/index.jsx#L113.

Reply:
yes, passed but the problem is we comment prop will be something like `<div>Comment</div>`  so we can't show it inside aria-label component.

Reply:
Right, I think for the comment case we have edge cases since comments don't have a name. I think we want to use the user's name that has made the comment post.